elrond-wasm/src/types/managed/big_uint_operators.rs

```diff
@@ -108,14 +108,14 @@ macro_rules! binary_assign_operator {
 
         impl<M: ManagedTypeApi> $trait<u32> for BigUint<M> {
             fn $method(&mut self, other: u32) {
-                let other_handle = M::managed_type_impl().bi_new(other as i64);
+                let other_handle = M::managed_type_impl().bi_new_handle(other as i64);
```

Comment:
Same.
We need to use something based on `bigIntSetInt64` here.
One of the temporary handles can be used.